hack/make-rules/test-e2e-node.sh

```diff
@@ -203,14 +203,16 @@ else
   # Refresh sudo credentials if needed
   if ping -c 1 -q metadata.google.internal &> /dev/null; then
```

Comment:
maybe ping the metadata server IP(169.254.169.254) instead of reworking the logic? The IP is the same on GCP and AWS

Reply:
I will leave `metadata.google.internal` as-is and add another check for `169.254.169.254`